src/Block.php

```diff
@@ -64,48 +113,69 @@ public function getImageUrl() {
     return FALSE;
   }
 
+  /**
+   * Renders the block into the block datastructure.
+   *
+   * If google analytics tracking is enabled, the corresponding JavaScript files
+   * are added.
+   *
+   * @see hook_block_view()
+   */
   public function render() {
     drupal_alter('share_light_options', $this->options);
 
-    // add tracking for GA if googleanalytics module is enabled
+    // Add tracking for GA if googleanalytics module is enabled
     // and share tracking is enabled (default: enabled)
     $tracking_enabled = module_exists('googleanalytics') && variable_get('share_light_tracking_enabled', 1);
     if ($tracking_enabled) {
       drupal_add_js(drupal_get_path('module', 'share_light') . '/tracking.js');
-      drupal_add_js(array('share_light' => array(
-        'share_url' => $this->getUrl(FALSE),
-      )), 'setting');
+      drupal_add_js(array(
+        'share_light' => array(
+          'share_url' => $this->getUrl(FALSE),
+        ),
+      ), 'setting');
     }
 
     $links = $this->channelLinks();
-
     foreach ($links as &$x) {
       $x['title'] = "<span>{$x['title']}</span>";
       $x['html'] = TRUE;
-      $x['attributes']['target'] = '_blank';
     }
 
     $block['subject'] = $this->options['subject'];
     $block['content'] = array(
       '#theme' => 'links',
       '#links' => $links,
       '#attributes' => array('class' => array('share-light')),
+      '#attached' => ['js' => [drupal_get_path('module', 'share_light') . '/hide_mobile_buttons.js']],
     );
+
     return $block;
   }
 
-  public static function from_current_path($options = array()) {
+  /**
+   * Creates a new Block instance.
```

Comment:
I think this should hint at the fact that the node is the thing that’s inferred from the current path.